scripts/validate_changelog_utils.py

```diff
@@ -7,15 +7,19 @@
 diff_file_start_regex = "^([\s]*)\@\@(.*)\@\@"
 pr_link_regex = "\[#\d+]\(https:\/\/github\.com\/mapbox\/mapbox-navigation-android\/pull\/\d+\)"
 
+
```

Comment:
That's not what I meant when saying we should adapt the checks...

Reply:
I didn't do anything with it))

Reply:
But there are changes in the diff...

Reply:
only autoformating